.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+### Expected behavior
+
```

Comment:
Perhaps give an example description of the expected behavior based on a real submission. 

> ### Expected behavior
> 
> Example (replace this example text with yours): I expected rsyslog to match on the hostname of the sending system and write a message to a file, but when I checked the real value using the DEBUG_Format template the hostname value is blank.

As an alternative, provide placeholder text that makes it clear you wish to have the user replace the placeholder text with their wording.

> ### Expected behavior
> 
> Insert your expectations here. The more detail you can provide, the better.